src/nvim/charset.c

```diff
@@ -799,32 +799,41 @@ unsigned int win_linetabsize(win_T *wp, char_u *line, colnr_T len)
   return (unsigned int)col;
 }
 
-/// Return TRUE if 'c' is a normal identifier character:
+/// Return true if 'c' is a normal identifier character:
```

Comment:
to be consistent with functions below next line should be joined with this (otherwise the `:` looks misplaced)